RELEASING.adoc

```diff
@@ -1,35 +1,57 @@
+The actual release process is triggered by the release admin.
+As opposed to the 1.x releases, the 2.0 release happens independent of the Spring Cloud release train.
+To simplify the process, perform this list of prerequisites before asking the release admin to run the release scripts.
 
-. Ensure that modules that shouldn't be released have the `maven-deploy-plugin` configuration set to `<skip>true</skip>`.
+. Ensure that modules that shouldn't be released are not included in the `<modules>` block in the `release` profile of `pom.xml`.
 For GA releases this should be configured under the `central` Maven profile.
 +
 [source,xml]
 ----
-<build>
-    <plugins>
-        <plugin>
-            <artifactId>maven-deploy-plugin</artifactId>
-            <configuration>
-                <skip>true</skip>
-            </configuration>
-        </plugin>
-    </plugins>
-</build>
+<profile>
+    <id>release</id>
+    <modules>
+        <!-- POM Modules-->
+        <module>spring-cloud-gcp-starters</module>
+        <module>spring-cloud-gcp-dependencies</module>
+
+        <!-- Code Modules-->
+        <module>spring-cloud-gcp-autoconfigure</module>
+        <module>spring-cloud-gcp-bigquery</module>
+        <module>spring-cloud-gcp-cloudfoundry</module>
+        <module>spring-cloud-gcp-core</module>
+        <module>spring-cloud-gcp-data-datastore</module>
+        <module>spring-cloud-gcp-data-firestore</module>
+        <module>spring-cloud-gcp-data-spanner</module>
+        <module>spring-cloud-gcp-logging</module>
+        <module>spring-cloud-gcp-pubsub</module>
+        <module>spring-cloud-gcp-pubsub-stream-binder</module>
+        <module>spring-cloud-gcp-security-iap</module>
+        <module>spring-cloud-gcp-storage</module>
+        <module>spring-cloud-gcp-secretmanager</module>
+        <module>spring-cloud-gcp-security-firebase</module>
+        <module>spring-cloud-gcp-vision</module>
+        ...
+    </modules>
+    ...
+</profile>
 ----
 
-. Wait for the release scripts to have been run by the release admin.
+. Ensure all Maven commands listed in `.kokoro/stage.sh` script can be executed successfully locally.
+You can add `-Dgpg.skip` in place of the actual GPG key locally.
+
+. Invoke the Kokoro `stage` job.
+Log in to OSS Sonatype and verify the contents of the staged package.
 
-. Make sure that the release tag was automatically added by the release process.
+. Invoke to the Kokoko `promote` job.
 
-. Make sure that the link:https://github.com/spring-io/initializr/blob/master/initializr-service/src/main/resources/application.yml[Spring Boot Initializr] is updated to support the new version of Spring Cloud GCP.
+. Make sure that the link:https://github.com/spring-io/start.spring.io/blob/master/start-site/src/main/resources/application.yml[Spring Boot Initializr] is updated to support the new version of Spring Cloud GCP.
```

Comment:
Does this happen automatically, or do we need to update Initializr config for every new version we release?

Reply:
I don't think it happens automatically - but the ranges specified should be updated as new Spring Boot versions appear. This way they know the tool isn't going to give users incompatible modules.

Reply:
Can ranges capture our future releases? And then, does Initializr automatically discover latest release in Maven Central or something?

Reply:
*Our* future releases aren't the problem as much as *their* future releases are. 

Initializer does not auto-discover anything - there is a URL (https://spring.io/project_metadata/spring-boot) that returns all "current" versions of Spring Boot and I don't know where that comes from, but AFAIK it's the only dynamic piece.

We would have to update Initializr when we do a release to bump a version.

Reply:
Can we put that into the instructions with more details like where the version would need to be updated?